### PR TITLE
HEADER names added

### PR DESCRIPTION
**Pull request by MWest2020**
NO merge conflicts, but IDK if ok. Formatter doesn't work

### CODE REVIEW

File: pwa/src/components/applications/applicationForm.tsx [hunk 1]
```diff
@@ -16,6 +16,7 @@ import APIContext from "../../apiService/apiContext";
 import LoadingOverlay from "../loadingOverlay/loadingOverlay";
 import { HeaderContext } from "../../context/headerContext";
 import { AlertContext } from "../../context/alertContext";
+import ApplicationsTable from "./applicationsTable";
```
Comment (lencodes) [inline pwa/src/components/applications/applicationForm.tsx]: I don't think we're actually using this import.

File: pwa/src/components/applications/applicationForm.tsx [hunk 2]
```diff
@@ -42,10 +43,10 @@ export const ApplicationForm: React.FC<ApplicationFormProps> = ({ id }) => {
 
   React.useEffect(() => {
     setHeader({
-      title: "Applications",
+      title: `Applications ${application ? application.name : ""}`,
```
Comment (lencodes) [inline pwa/src/components/applications/applicationForm.tsx]: This should be singular (we're currently in a single **application** and not in applications).

File: pwa/src/components/attributes/attributeForm.tsx [hunk 3]
```diff
@@ -275,7 +275,7 @@ export const AttributeForm: React.FC<AttributeFormProps> = ({ attributeId, entit
                     </div>
                     <br />
                     <div className="row">
-                      <div className="col-6">
+                      {/* <div className="col-6">
```
Comment (lencodes) [inline pwa/src/components/attributes/attributeForm.tsx]: This shouldn't be commented out.

File: pwa/src/pages/entities/[entityId]/attributes/[attributeId].tsx [hunk 4]
```diff
@@ -1,5 +1,5 @@
 import * as React from "react";
-import AttributeForm from "../../../../components/attributes/attribuutForm";
+import AttributeForm from "../../../../components/attributes/attributeForm";
```
Comment (lencodes) [inline pwa/src/pages/entities/[entityId]/attributes/[attributeId].tsx:2]: 👍🏼

**Review by lencodes — CHANGES_REQUESTED** [review]
Two global things we should add to this pull request:

1. Currently, the `HeaderContext` accepts as the `title: string` a string, if we make this a `title: string | JSX.Element` we can change your `setHeading` syntax throughout all the files you've updated to (e.g.): 

```
setHeader({
  title: <>Application {application && <i>{application.name}</i>}</>,
  subText: "Manage your applications here",
});
```

This way, its italic and we can remove the `? <> : <>` syntax.

Note this would mean updating the `headerContext.ts` props to:

```
export interface HeaderProps {
  title: string | JSX.Element;
  subText: string;
}
```

2. Since we've added breadcrumbs, the `subText` in the `setHeader()` function is actually **never used**. So we can remove this by (1.) removing the `subText` prop from the `HeaderContext` and (2.) removing the `subText` prop from every `setHeader()` call (throughout all files, not just the ones changed in this pull request).

Edit: this would mean that the final implementation would be (e.g.):

```
setHeader({
  title: <>Application {application && <i>{application.name}</i>}</>,
});
```

File: pwa/src/components/applications/applicationForm.tsx (hunk 1 above)
Comment (MWest2020) [reply pwa/src/components/applications/applicationForm.tsx]: check

**Pull request closed without merging**